examples/uplay.c

```diff
@@ -537,6 +536,8 @@ static void uplay_start(struct upump *upump)
                              UPROBE_LOG_VERBOSE, "wsrc_x"),
             src_out_queue_length);
 
+    uprobe_throw(uprobe_main, NULL, UPROBE_THAW_UPUMP_MGR);
```

Comment:
In this case I'll drop that commit because trickp / play pipes are not used in the deported pipe.